docs/guides/documentdb/rotate-authentication/rotate-authentication.md

```
@@ -0,0 +1,162 @@
+title: Rotate Authentication DocumentDB
+menu:
+  docs_{{ .version }}:
+    identifier: dc-rotate-authentication-details
+    name: Rotate Authentication
+    parent: dc-rotate-authentication
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)